.github/CODEOWNERS

```
@@ -8,6 +8,6 @@
 /.github/                                                                         @dotnet/aspnet-build
 /src/                                                                             @dotnet/aspnet-blazor-eng
 /eng/                                                                             @dotnet/aspnet-build
-/eng/common/                                                                      @dotnet-maestro-bot
+/eng/common/                                                                      @dotnet-maestro-bot @dotnet/aspnet-build
 /eng/Versions.props                                                               @dotnet-maestro-bot
 /eng/Version.Details.xml                                                          @dotnet-maestro-bot
```

The idea is that @wtgodbe and I care about everything in eng/ except eng/common/ files. But, the bot needs to care about the Versions*.* files.
```suggestion
/eng/common/                                                                      @dotnet-maestro-bot
/eng/Versions.props                                                               @dotnet-maestro-bot @dotnet/aspnet-build
/eng/Version.Details.xml                                                          @dotnet-maestro-bot @dotnet/aspnet-build
```